extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts

```diff
@@ -13,7 +13,7 @@ const webviewId = 'myWebview';
 
 const testDocument = join(vscode.workspace.rootPath || '', './bower.json');
 
-suite('Webview tests', () => {
+suite.skip('Webview tests', () => {
```

Comment:
Hey looks like web views are broken under electron due to `Prevented webview attach`. I'm looking into it but wonder if you have any idea what that may be?

Reply:
Pushed fix with c8123dee962847abcbd988d8c180ae1dec0b56d5 and re-enabled the tests